jobs/admin.py

```diff
@@ -22,7 +22,7 @@ def make_published(modeladmin, request, queryset):
 
 
```

Comment:
Please change the import statements to use django.conf
```
from djangogirls.settings import JOBS_EMAIL_USER, JOBS_EMAIL_PASSWORD
from djangogirls.settings import MEETUPS_EMAIL_PASSWORD, MEETUPS_EMAIL_USER
```

change to
```
from django.conf import settings
```

If you don't want to change the code to refer to e.g. settings.MEETUPS_EMAIL_PASSWORD, just add under the import statements
```
MEETUPS_EMAIL_PASSWORD = settings.MEETUPS_EMAIL_PASSWORD
```
etc

This way the correct settings will be imported irrespective of which particular settings file is being used.